migration/sql-files/028-iterations-parent-path-ltree.sql

```diff
@@ -0,0 +1,22 @@
+CREATE EXTENSION IF NOT EXISTS "ltree";
+
+-- Rename parent_id column
+ALTER TABLE iterations RENAME parent_id to path;
+
+-- Need to convert the path column to text in order to
+-- replace non-locale characters with an underscore
+ALTER TABLE iterations ALTER path TYPE text USING path::text;
+
+-- Need to update values of Iteration's' ParentID in order to migrate it to ltree
+-- Replace every non-C-LOCALE character with an underscore
+UPDATE iterations SET path = regexp_replace(path, '[^a-zA-Z0-9_\.]', '_', 'g');
```

Comment:
What happens to existing multi-level child iterations ?

Example:
`i1` --> `i2` --> `i3`

As per previous model , parent of `i3` would have just stored `i2` .
In the new model, `parent_path` of `i3` would be `i2` which is incorrect. It should be `i1.i2` - 

Does the production database table have such ( multi-level children ) data? If it's all 1-level , we are lucky ;)